src/pages/search/model.ts

```diff
@@ -1,39 +1,13 @@
-import type { Card, User } from '@box/api';
-import {
-  combine,
-  createEffect,
-  createEvent,
-  createStore,
-  sample,
-} from 'effector-root';
-import { internalApi } from '@box/api';
-import { model } from '@box/features/search-bar';
+import { combine, createEvent } from 'effector-root';
 
-export const $searchResultCardList = createStore<Card[]>([]);
-export const $searchResultUserList = createStore<User[]>([]);
-export const $searchCardsCount = createStore<number>(0);
-export const $searchUsersCount = createStore<number>(0);
+import { searchModel } from '../../features/search-bar';
```

Comment:
Я бы не прекращал юзать альясы для верхнеуровневых слоев
```suggestion
import { searchModel } from '@box/features/search-bar';
```

Reply:
уф(

Reply:
идэешечка не распознаёт, руками приходится(

Reply:
ну шо поделать(

Reply:
Мне кажется альясы можно расставить как-то стопроц

Reply:
фиксанул, кажется.